src/dotnet/APIView/APIView/AttributeConstructArgAPIV.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.CodeAnalysis;
+using System.Collections.Generic;
+
+namespace APIView
+{
+    public class AttributeConstructArgAPIV
+    {
+        public bool IsNamed => Name != null;
+        public string Name { get; set; }
+        public string Value { get; set; }
+
+        public AttributeConstructArgAPIV() { }
+
+        public AttributeConstructArgAPIV(TypedConstant value)
+        {
+            if (value.Type.SpecialType == SpecialType.System_String)
+                this.Value = "\"" + value.Value.ToString() + "\"";
+            else
+                this.Value = value.Value.ToString();
+        }
+
+        public AttributeConstructArgAPIV(KeyValuePair<string, TypedConstant> pair)
```

Comment:
Is this one used? And why?

Reply:
This is the format the named arguments come in, so I kept the KeyValuePair for the constructor. I could also slit the input up into the name and value if that's preferable

Reply:
Hm, change it to take name and value instead, it would be slightly cleaner.